CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 # `wasm-bindgen` Change Log
 --------------------------------------------------------------------------------
 
+## Unreleased
+
+### Fixed
+
+- Fixed JS keyword-like identifiers not being handled correctly for imports and exports.
+  [#4329](https://github.com/rustwasm/wasm-bindgen/pull/4329)
```

Comment:
Considering this is a breaking change, even if an acceptable one, this needs to be expanded.
Mention at least all the things that are now a compile error that weren't before and why they are now.